Gemfile

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
 # Specify your gem's dependencies in technologies-list-maple.gemspec
 gemspec
 
-gem "rake", "~> 13.0"
+gem 'rake', '~> 13.0'
 
-gem "rspec", "~> 3.0"
+gem 'rspec', '~> 3.0'
 
-gem "rubocop", "~> 1.21"
+gem 'rubocop', '~> 1.21'
+
+gem 'byebug', platforms: %i[mri mingw x64_mingw]
```

Comment:
should they only be in development? Otheriwse they will install this in prod too